docs/lifecycle/lifecycle.rst

```diff
@@ -198,12 +303,8 @@ reviews.
 Disposition of Existing Projects
 --------------------------------
 
-OpenDaylight, OPNFV, FD.io, and ONAP are in state TAC. PNDA and SNAS are in
-state Non-TAC. Tungsten Fabric was `conditionally inducted by the Board
-<tf-condit-induct-email_>`__ as Non-TAC and should do an LFN Entry Review once
-the process is established.
-
-.. _tf-condit-induct-email: https://lists.lfnetworking.org/g/TAC/message/250
+OpenDaylight, OPNFV, FD.io, and ONAP are in state TAC. PNDA, SNAS, and Tungsten
+Fabric are in state Incubation (?).
```

Comment:
We need to resolve where the current non-TAC projects fall in the new structure.  Are they all Incubation?

Reply:
put all current non-TAC projects in as incubation in the next commit